akka-remote/src/main/scala/akka/remote/transport/netty/NettySSLSupport.scala

```diff
@@ -127,6 +127,7 @@ private[akka] object NettySSLSupport {
           val sslEngine = context.createSSLEngine
           sslEngine.setUseClientMode(true)
           sslEngine.setEnabledCipherSuites(settings.SSLEnabledAlgorithms.toArray)
+          sslEngine.setEnabledProtocols(settings.SSLProtocol.toArray)
```

Comment:
Does this change the default behavior or with no config it just picks up the default list of protocols?